### PR TITLE
[Merged by Bors] - switched to dev and added some timeouts

### PR DESCRIPTION
**Pull request by fhennig**
# Description
test: https://ci.stackable.tech/view/05%20Documentation%20Tests/job/superset-operator-documentation-test/6/

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Review by maltesander — APPROVED** [review]
LGTM!

File: docs/modules/superset/examples/getting_started/getting_started.sh [hunk 1]
```diff
@@ -67,17 +69,27 @@ echo "Creating Superset cluster"
...
 # tag::wait-supersetdb[]
 time kubectl wait supersetdb/simple-superset \
   --for jsonpath='{.status.condition}'=Ready \
   --timeout 600s
 # end::wait-supersetdb[]
 
-sleep 5
+for (( i=1; i<=15; i++ ))
+do
+  echo "Waiting for DruidCluster to appear ..."
```
Comment (maltesander) [inline docs/modules/superset/examples/getting_started/getting_started.sh:83]:
```suggestion
  echo "Waiting for SupersetCluster to appear ..."
```

**Review by maltesander — CHANGES_REQUESTED** [review]

File: docs/modules/superset/examples/getting_started/getting_started.sh.j2 [hunk 2]
```diff
@@ -67,17 +69,27 @@ echo "Creating Superset cluster"
...
 # tag::wait-supersetdb[]
 time kubectl wait supersetdb/simple-superset \
   --for jsonpath='{.status.condition}'=Ready \
   --timeout 600s
 # end::wait-supersetdb[]
 
-sleep 5
+for (( i=1; i<=15; i++ ))
+do
+  echo "Waiting for DruidCluster to appear ..."
```
Comment (maltesander) [inline docs/modules/superset/examples/getting_started/getting_started.sh.j2:83]:
```suggestion
  echo "Waiting for SupersetCluster to appear ..."
```

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**